calico_containers/tests/unit/calicoctl_test.py

```diff
@@ -25,11 +29,12 @@
 from calico_ctl.node import validate_arguments as node_validate_arguments
 from calico_ctl.pool import validate_arguments as pool_validate_arguments
 from calico_ctl.profile import validate_arguments as profile_validate_arguments
```

Comment:
All of these imports should probably just be called like `<module>.validate_arguments` now, rather than importing each  one individually in addition to the module and naming it pretty much the same thing.